scripts/stac/imagery/tests/capture_area_test.py

```diff
@@ -252,9 +260,11 @@ def test_should_make_compliant_capture_area(subtests: SubTests) -> None:
         ),
     ]
 
-    capture_area = merge_polygons(polygons, 0.1)
+    merged_polygons = merge_polygons(polygons, 0.1)
+    mp = cast(MultiPolygon, merged_polygons)
```

Comment:
We might need to do that, to verify

Reply:
It was the PyCharm default linter requiring it but Pylint is fine with it so I removed the cast.